app/models/daily_progress.rb

```diff
@@ -0,0 +1,5 @@
+class DailyProgress < ActiveRecord::Base
+  belongs_to :practice
+  validates  :practice_id, presence: true
+  validates  :done_at,     presence: true
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.